libs/coin-framework/src/bridge/jsHelpers.test.ts

```diff
@@ -37,4 +38,89 @@ describe("jsHelpers", () => {
       });
     });
   });
+
+  describe("makeSync", () => {
+    it("returns a function to update account that give a new instance of account", done => {
+      // Given
+      const account = createAccount("12");
+
+      // When
+      const accountUpdater = makeSync({
+        getAccountShape: (_accountShape: AccountShapeInfo) => Promise.resolve({} as Account),
+      })(account, {} as SyncConfig);
+
+      accountUpdater.subscribe(updater => {
+        // Then
+        expect(updater(account)).not.toBe(account);
+        done();
```

Comment:
We could use async/await and firstValueFrom rxjs operator to match the rest of the codebase or atleast destroy the subscription

Reply:
Resolving as it's treated